src/components/TrashNote/TrashNote.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { useNote } from "../../contexts/Note/NoteContext";
+import styles from "./TrashNote.module.css";
+
+export default function TrashNote({ note }) {
+  const { title, notes } = note;
+  const [options, setOptions] = useState(false);
+  const { noteState, setNote, noteDispatch, archiveNote, RestoreArchivedNote } =
+    useNote();
+
+  const { color, label, day, timestamp } = note;
```

Comment:
Can combine terms of line number 7 &12 into a single destructured one!!

Reply:
Yes, My bad. Will correct it.